.github/workflows/ci.yaml

```diff
@@ -16,13 +16,13 @@ jobs:
       test-chunks:  ${{ steps['set-test-chunks'].outputs['test-chunks'] }}
       test-chunk-ids: ${{ steps['set-test-chunk-ids'].outputs['test-chunk-ids'] }}
     steps:
-      - uses: webfactory/ssh-agent@v0.5.4
+      - uses: webfactory/ssh-agent@v0.7.0
         with:
           ssh-private-key: ${{ secrets.SSH_GITHUB_SOLIDOC }}
       - name: Checkout repository
-        uses: actions/checkout@v2.4.0
+        uses: actions/checkout@v3
```

Comment:
Does @v3 get the latest version?

Reply:
I could not find any minor version on the docs; every example I have seen uses only `v3`. It's weird because the repo release says that the latest is 2.5.0 https://github.com/actions/checkout, not sure if I understood how the actions versions works

Reply:
The way the releases show up is a bit weird. I don't know why it shows v2.5.0 as the latest. If you go here, you see that 3.1.0 is actually the latest: https://github.com/actions/checkout/releases. Maybe 2.5.0 is just the latest for v2.

Reply:
I changed to the latest both checkup and setup-node actions. I had tried with 3.1.x or 3.x.x but it didn't work